Saas/Publish-PerTenantExtensionApps.ps1

```diff
@@ -42,6 +42,8 @@ function Publish-PerTenantExtensionApps {
         $appFiles,
         [ValidateSet('Add','Force')]
         [string] $schemaSyncMode = 'Add',
+        [ValidateSet(' ','Current version','Next minor version','Next major version')]

```

Comment:
Please use empty string as default instead of a space.

Reply:
Fixed